tests/behaviour/typeql/steps.rs

```diff
@@ -88,4 +92,78 @@ generic_step_impl! {
         assert!(result.is_err());
         assert!(result.unwrap_err().contains(&exception));
     }
+
+    #[step(expr = "get answers of typeql match")]
+    async fn get_answers_of_typeql_match(context: &mut Context, step: &Step) {
+        let parsed = parse_query(step.docstring().unwrap());
+        assert!(parsed.is_ok());
+        let stream = context.transaction().query().match_(&parsed.unwrap().to_string());
+        assert!(stream.is_ok());
+        let res = stream.unwrap().try_collect::<Vec<_>>().await;
+        assert!(res.is_ok());
+        context.answer = res.unwrap();
+    }
+
+    #[step(expr = "answer size is: {int}")]
+    async fn answer_size(context: &mut Context, step: &Step, expected_answers: usize) {
+        assert_eq!(
+            context.answer.len(),
+            expected_answers,
+            "The number of identifier entries (rows) should match the number of answers, \
+            but found {0} identifier entries and {1} answers.",
+            expected_answers,
+            context.answer.len()
+        );
```

Comment:
Love this, very thorough. One suggestion would be to use string interpolation where we can, and only use the indices (`{1}`, e.g.) when necessary.

Reply:
Refactored